metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -0,0 +1,124 @@
+
+  Scenario Outline: Objects to which at least released study links should be accessible, and vice versa
+
+    # Create the common taxonomy
+    When I request POST taxonomies with 9606 for ID, Homo Sapiens for name and NONE for ancestors
+    Then set the URL to TAXONOMY
+    And the response code should be 201
+
+    # Create the common reference sequences
+    Given I request POST /reference-sequences with JSON-like payload:
+    """
+          "name": "GRCh37",
+          "patch": "p2",
+          "accessions": ["GCA_000001407.3", "GCF_000001407.14"],
+          "type": "ASSEMBLY",
+          "taxonomy": "TAXONOMY"
+    """
+    Then set the URL to REFERENCE_SEQUENCE
+    And the response code should be 201
+
+    # Create two studies with the release date of "yesterday"
+    Given I create a study with TAXONOMY for taxonomy and Study1 for accession
+    Then set the URL to STUDY1
+    And the response code should be 201
+    Given I create a study with TAXONOMY for taxonomy and Study2 for accession
+    Then set the URL to STUDY2
+    And the response code should be 201
+
+    # Create two analyses: Study1 → Analysis1 and Study2 → Analysis2
+    Given I create an analysis with Analysis1 for accession, REFERENCE_SEQUENCE for reference sequence and STUDY1 for study
+    Then set the URL to ANALYSIS1
+    And the response code should be 201
+    Given I create an analysis with Analysis2 for accession, REFERENCE_SEQUENCE for reference sequence and STUDY2 for study
+    Then set the URL to ANALYSIS2
+    And the response code should be 201
+
+    # Create three files
+    Given I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File1",
+        "version": 1
+        },
+      "hash": "Hash1",
+      "name": "File1",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    Then the response code should be 201
+    And set the URL to FILE1
+
+    Given I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File2",
+        "version": 1
+        },
+      "hash": "Hash2",
+      "name": "File2",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    Then the response code should be 201
+    And set the URL to FILE2
+
+    Given I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File3",
+        "version": 1
+        },
+      "hash": "Hash3",
+      "name": "File3",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    Then the response code should be 201
+    And set the URL to FILE3
+
+    # Assign files to analyses: Analysis1 → File1, Analysis2 → File2, Analysis1,2 → File3
+    When I request PATCH ANALYSIS1 with list FILE1,FILE3 of files
+    Then the response code should be 2xx
+    When I request PATCH ANALYSIS2 with list FILE2,FILE3 of files
+    Then the response code should be 2xx
+
+    # Set release dates for studies
```

Comment:
This block should be:

1. When
2. Then
3. When
4. Then

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/2da2671086076bae4c89bb2b6f79b4363089ef75.